### ISSUE TITLE
Change to a React App

### ISSUE THREAD

**Issue by ivanoats** [orcasound/orcamap#18]
As we now have developers with ReactJS knowledge on the volunteer team and
In order to facilitate integrating the Orcamap into Orcasite,
And improve the modularity, reusability, and developer experience of the code
We should convert this app to a React app, while it is still small and manageable, before it gets to be too big of a chore.

**Comment by aloksingh3112** [comment]
Yes @ivanoats looks like a good idea right now the codebase has got a little old and still a lot of improvements can be done.

**Comment by ivanoats** [comment]
working in a new repo here: https://github.com/orcasound/orcamap-react

**Comment by cpsarason** [comment]
Sounds sgood to me, asgood a time as any for finally pick up some react
skills!

Best
C

---
+1.206.992.1866
@cpsarason | cpsarason.github.io

**Comment by ivanoats** [comment]
Thank you Christian! 😄

**Comment by Dhiraj240** [comment]
@ivanoats Sir, i am directly applying on gsoc website and have put more focus over this project.
One doubt is should we include every feature of ```orcacamp``` to new react setup.

**Comment by ivanoats** [comment]
Definitely only implement features in the React codebase. Please review the history in the #orcamap-react Slack channel for important architecture discussions.

**Comment by Dhiraj240** [comment]
Alright, thank you for the information and are the previous contributions in organisation counts?

**Comment by ivanoats** [comment]
Check the orcasound/orcagsoc repo for what is needed in proposals and how we score proposals.

**Comment by Dhiraj240** [comment]
Alright, I would contribute even after 31 march.

**Comment by Dhiraj240** [comment]
@ivanoats Sorry, but I would not be able to submit the proposal this year for this project as I am out of time.
Although I submitted the proposal for project 3 streaming software optimization.
